charts/bulk-scan-orchestrator/values.preview.template.yaml

```diff
@@ -13,6 +13,7 @@ java:
     DELETE_ENVELOPES_DLQ_MESSAGES_ENABLED: "true"
     DELETE_ENVELOPES_DLQ_MESSAGES_CRON: "0 * * * * *"
     DELETE_ENVELOPES_DLQ_MESSAGES_TTL: "10s"
+    TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
```

Comment:
unfortunately, this will have to be a separate pr as it is a config change. only minor one.

but if we're happy to stick this in together @nitinprabhu then let's bump chart version here too and we are all good

Reply:
Yes as commented on pr earlier this should go as a separate pr and bump chart version including flux repo

Reply:
Everything done - PR-660